packages/frontend-web/src/app/scenes/Tables/components.tsx

```diff
@@ -39,7 +37,7 @@ function maybeS(val: number) {
 }
 
 export class MagicTimestamp extends React.Component<IMagicTimestampProps> {
-  timeoutId?: Timer;
+  timeoutId?: any;
```

Comment:
sigh :)

Reply:
Sorry, I don't know what to do with this one.  The setTimer return value is a completely different type for node.js and on the browser.  And the former type isn't even available when building for the browser, so can't do an either/or type...

Reply:
oop, just spotted ReturnType...
ReturnType<typeof setTimeout>;  
I'll see if that works...

Reply:
If you need a generic type Object, prefer `{}` instead of `any`.